packages/sites/src/site-excerpt-constants.ts

```diff
@@ -21,6 +21,7 @@ export const SITE_EXCERPT_REQUEST_FIELDS = [
 	'user_interactions',
 	'lang',
 	'site_owner',
+	'is_a8c',
```

Comment:
Hey @p-jackson 👋🏼🙂 

Just would like to check with you what do you think about adding `is_a8c` field to `SITE_EXCERPT_REQUEST_FIELDS`, more specifically in relation to your code comment:

> // Performance-optimized request for lists of sites.
> // Don't add more fields because you will make the request slower.

Context: We are aiming to use the new `is_a8c` field to identify A8C-related sites (instead of relying on `site_owner` field): DOTCOM-10699-linear-issue.

Thank you.

Reply:
Sorry for the slow reply.

It is too bad that we have to add so many fields to the "site excerpt" data type. The whole idea is that we should only have to fetch a small number of fields in order to render the skeleton of the sites dashboard. But we keep adding more features to the dashboard so it's sort of unavoidable 😅

But if the `is_a8c` field is needed to determine site visibility, then it's sort of unavoidable that it's needed.

Since the "Include A8C sites" filter is in a dropdown, it sort of implies the `is_a8c` filtering could be done server side? In which case we wouldn't need to add it to the "site excerpt" type. But if that wasn't doable, then yeah, adding it to the `SITE_EXCERPT_REQUEST_FIELDS` list makes sense.

Reply:
Thank you, Philip! No worries about later response. 🙂 

> Since the "Include A8C sites" filter is in a dropdown, it sort of implies the is_a8c filtering could be done server side? In which case we wouldn't need to add it to the "site excerpt" type. But if that wasn't doable, then yeah, adding it to the SITE_EXCERPT_REQUEST_FIELDS list makes sense.

I think this is doable indeed. The reason we used the current approach is that initially we implemented `is_a8c` for the need of the Studio app. Only later, when I was working on this Calypso PR, I noticed the `SITE_EXCERPT_REQUEST_FIELDS`.

The change to add the filtering to backend will require a bit more work, so I have created a new task that we could look into as maintenance: STU-503-linear-issue.